app/views/shared/_pii_review.html.slim

```diff
@@ -13,7 +13,8 @@
   .h4.bold.ico-absolute.ico-absolute-success #{pii[:ssn]}
 
   .h6.mt3 = t('idv.messages.phone.phone_of_record')
-  .h4.bold.ico-absolute.ico-absolute-success #{pii[:phone]}
+  - phone = pii[:phone].phony_formatted(format: :international, normalize: :US, spaces: ' ')
```

Comment:
Looks like this might need a `.try` or a `&.` or something based on some errors in the specs